NetflixClone/Classes/Models/Movie.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  Movie.swift
+//  NetflixClone
+//
+//  Created by Joseph Ching on 2022-09-27.
+//
+
+import Foundation
+
+struct Movie: Codable, Hashable {
+    let id: Int
+    let adult: Bool?
+    let backdropPath: String?
+    let genreIDS: [Int]?
+    let originalLanguage, originalTitle, overview: String?
+    let popularity: Double?
+    let posterPath, releaseDate, title: String?
+    let video: Bool?
+    let voteAverage: Double?
+    let voteCount: Int?
+
+    enum CodingKeys: String, CodingKey {
+        case adult
+        case backdropPath = "backdrop_path"
+        case genreIDS = "genre_ids"
+        case id
+        case originalLanguage = "original_language"
+        case originalTitle = "original_title"
+        case overview, popularity
+        case posterPath = "poster_path"
+        case releaseDate = "release_date"
+        case title, video
+        case voteAverage = "vote_average"
+        case voteCount = "vote_count"
+    }
+}
+
+struct Item: Sendable, Hashable {
```

Comment:
Is it needed? if yes, we can add it as a nested type to movie
extension Movie { 
     struct Item {
          let name: String
          let idNumber: Int
          let rating: Double
          let tags: [String]
     }
}